backend/index.js

```diff
@@ -7,15 +8,46 @@ const io = require("socket.io")(server);
 const map = require("../public/assets/map/map.js"); // eslint-disable-line
 let players = require("./players");
 
-let timePast = Date.now();
+const grid = new PF.Grid(map);
 
-const canGo = ({ x, y }) => {
-  const tile = map[y][x];
+const finder = new PF.AStarFinder({
+  allowDiagonal: true,
+});
 
-  // TODO (#50): add check if there is no other player at the moment
+setInterval(() => {
+  players = players.map((player) => {
```

Comment:
It would be nice to wrap this movement/pathfinding stuff into some function to make it more clear what happens on each interval iteration.